tests/integration/helpers/react.tsx

```diff
@@ -151,7 +153,10 @@ export function createAppProvidersWrapper(
           }
           coordinateFormatStore={persistedCoordinateFormatStore}
           trackStore={persistedTrackStore}>
-          {children}
+          {/* We don't use navigation in our tests, but we set up the context. */}
+          <NavigationContainer ref={rootNavigationRef}>
```

Comment:
wondering if this should be creating its own `rootNavigationRef` as opposed to the singleton that the actual app uses. seems like using this is prone to issues around test isolation

Reply:
Removed in 1b7f17428b820a17267edb0f02b3aa26df835d2f.